istio-control/istio-discovery/templates/enable-mesh-mtls.yaml

```diff
@@ -1,4 +1,22 @@
+# Destination rule to disable (m)TLS when talking to API server, as API server doesn't have sidecar.
```

Comment:
just a TODO note - in the new installer I am keen to separate the CR creation from the rest of the system (so they can be created ahead of the deployments).

So we would have 3 steps:
1. install systemwide CRDs once
1. install CR for the feature
1. install manifest for the feature

Reply:
Ack - maybe we can record this in a doc or wiki.